server/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -88,6 +92,7 @@
   private static final String OBJECTMAPPER_ATTRIBUTE = "org.apache.druid.proxy.objectMapper";
 
   private static final int CANCELLATION_TIMEOUT_MILLIS = 500;
+  private static final long serialVersionUID = 2026655693637894524L;
```

Comment:
It's because I'm using the transient keyword below and the class is serializable ([documentation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html)) and basically to satisfy LGMT